app/controllers/discourse_automation/automations_controller.rb

```diff
@@ -2,12 +2,27 @@
 
 module DiscourseAutomation
   class AutomationsController < ApplicationController
-    before_action :ensure_admin
+    before_action :ensure_admin, except: [:post_checked]
+    before_action :ensure_logged_in, only: [:post_checked]
```

Comment:
Why do we need `ensure_logged_in`? I think `ensure_admin` implicitly requires a user to be logged in since it requires there's a admin user making the requests